documentation/src/main/asciidoc/topics/con_xsite_expiration.adoc

```diff
@@ -1,7 +1,10 @@
 [id='xsite_expiration-{context}']
-= Expiration and Cross-Site Replication
-{brandname} expiration controls how long entries remain in the cache.
+= Expiration with Cross-Site Replication
+Expiration removes cache entries based on time.
+{brandname} provides two ways to configure expiration for entries:
 
-* `lifespan` expiration is suitable for cross-site replication. When entries reach the maximum lifespan, {brandname} expires them independently of the remote sites.
+`lifespan`:: Sets the maximum amount of time that entries can exist. +
+When you set `lifespan` with cross-site replication, {brandname} clusters expire entries independently of remote sites.
 
-* `max-idle` expiration does not work with cross-site replication. {brandname} cannot determine when cache entries reach the idle timeout in remote sites.
+`max-idle`:: Specifies how long entries can exist based on read or write operations in a given time period. +
+When you set a `max-idle` with cross-site replication, {brandname} clusters send touch commands during replication to coordinate idle timeout with remote sites.
```

Comment:
@wburns Does this description seem adequate? I didn't want to get overly complex about the touch commands. I know if they return null then the entry is expired. But I'm not sure we need to say that. 

Is there any performance penalty that comes from latency with touch commands?

Reply:
I believe the touch commands are sent on access/read. IMO, the word "replication" implies some kind of write operation and should be avoided.

Performance is expected to be worse. You will go from simple RAM access (best case) to a remote site round-trip (ouch!)... There is no free lunch here :) The good part is the touch command isn't required in every read...